Plots/Panels/NCL_panel_10.py

```diff
@@ -0,0 +1,258 @@
+
+# Format axes for zonal average plot
+# Use geocat.viz.util convenience function to set axes limits & tick values
+gvutil.set_axes_limits_and_ticks(ax2,
+                                 xlim=[-0.6, 0.9],
+                                 ylim=[0, 1.55 * 1e16],
+                                 xticks=np.arange(-0.3, 0.7, 0.3),
+                                 yticks=np.linspace(0, 1.55 * 1e16, 7),
+                                 xticklabels=['-0.30', '', '0.30', ''],
+                                 yticklabels=np.arange(0, 181, 30))
+
+# Use geocat.viz.util convenience function to add minor and major tick lines
+gvutil.add_major_minor_ticks(ax2,
+                             x_minor_per_major=1,
+                             y_minor_per_major=3,
+                             labelsize=12)
+
+# Remove tick marks on left side of ax2
+ax2.tick_params('y', which='both', left=False)
+
+# Use geocat.viz.util convenience function to add titles
+gvutil.set_titles_and_labels(ax2,
+                             maintitle="Zonal Ave",
+                             maintitlefontsize=12)
+
+# Plot zonal average
+ax2.plot(mean, times, linewidth=0.5, color='black')
+
+plt.show()
+
+##############################################################################
+# Define helper function to create the four subplots
+
+
+def make_subplot(fig, gridspec, xlim):
+    # Create axes for the contour plot and the zonal average plot
+    ax1 = fig.add_subplot(gridspec[0])
+    ax2 = fig.add_subplot(gridspec[1])
+
+    # Draw contour lines
+    ax1.contour(lon,
+                times,
+                chi,
+                levels=np.arange(-12, 13, 2),
+                colors='black',
+                linestyles='solid',
+                linewidths=.5)
+
+    # Draw filled contours, save the mappable to create colorbar later
+    cf = ax1.contourf(lon,
+                      times,
+                      chi,
+                      levels=np.arange(-12, 13, 2),
+                      cmap=gvcmaps.BlWhRe)
+
+    # Use geocat.viz.util convenience function to add longitude tick labels
+    gvutil.add_lat_lon_ticklabels(ax1)
+
+    # Use geocat.viz.util convenience function to set axes limits & tick values
+    gvutil.set_axes_limits_and_ticks(ax1,
+                                     xlim=xlim,
+                                     ylim=[0, 1.55 * 1e16],
+                                     xticks=np.arange(xlim[0], xlim[1], 30),
+                                     yticks=np.linspace(0, 1.55 * 1e16, 7),
+                                     yticklabels=[])
+
+    # Use geocat.viz.util convenience function to add minor and major tick lines
+    gvutil.add_major_minor_ticks(ax1,
+                                 x_minor_per_major=2,
+                                 y_minor_per_major=3,
+                                 labelsize=10)
+
+    # Remove tick marks on right side of ax1
+    ax1.tick_params('y', which='both', right=False)
+
```

Comment:
I'm not sure if you were able to find another way to remove the degree symbol other than with the `ax.xaxis.set_major_formatter()` used in the past (looking at the first plot), but I would add that feature to the 4 panel plot as well for NCL consistency

Reply:
Fixed! Thanks for pointing it out